WWDC/LiveObserver.swift

```diff
@@ -221,11 +221,13 @@ private final class CloudKitLiveObserver: Logging {
                                                    subscriptionID: specialLiveEventsSubscriptionID,
                                                    options: options)
 
-            database.save(subscription) { _, error in
+            database.save(subscription) { [weak self] _, error in
+                guard let self = self else { return }
+
                 if let error = error {
-                    log.error("Error creating subscriptions: \(String(describing: error), privacy: .public)")
+                    self.log.error("Error creating subscriptions: \(String(describing: error), privacy: .public)")
```

Comment:
Must've been copy paste, the regex converter I built after I completed the migration by hand wouldn't have made that mistake 🤣